### ISSUE TITLE
App crashes on launch after 2.2.1 Beta 2 update [bug]

### ISSUE THREAD

**Issue by yvesquadros** [noah-nuebling/Mac-Mouse-Fix#327]
## Description

2.2.1 Beta 1 worked fine, but Beta 2 crashes on launch. Background process is still active, though. I'll try to remove, reboot and reinstall. 

EDIT: Rebooting, removing, reinstalling didn't help either. Back to Beta 1

## Steps to reproduce

1 Updated to Beta 2 via the Check for updates
2 Launched the app from the Dock
3 Crashes.

## Context

Mac Mouse Fix Version: 2.2.1 Beta 2
macOS Version: macOS 12.5 (21G72) MacBookPro12,1
Mouse Model: Microsoft Wireless Mouse 5000   Wireless Keyboard 3000 v2.0 USB Device: Microsoft® 2.4GHz Transceiver v7.0

## Additional Comments

Mac Mouse Fix is amazing, thank you for releasing it and maintaining it so actively. I'm not a developer, but if there is anything I can contribute as a designer or with translating the app to Brazilian Portuguese, let me know. Thanks.

---

### Console Logs

https://pastebin.com/JJ6wx4Mt

### Crash Reports

https://pastebin.com/NiD9UeTQ

**Comment by noah-nuebling** [comment]
Hi Yves, thank you so much for the kind words and for the detailed feedback! If you ever have any ideas or criticisms for the UI I'm always very interested in that. If you make something alone, it's easy to develop blindspots.

From the crash report it looks like what you're experiencing is a bug in Apple's ServiceManagement API. 

I think that because of these lines in the crash report you attached to your report:
```
Symbol not found: (_OBJC_CLASS_$_SMAppService)
Referenced from: '/Applications/Mac Mouse Fix.app/Contents/MacOS/Mac Mouse Fix'
Expected in: '/System/Library/Frameworks/ServiceManagement.framework/Versions/A/ServiceManagement'
```

I think this will be fixed in a later macOS Beta. If not I'll look into it again.

Hope that helps a little. Let me know if you have any more questions.

**Comment by noah-nuebling** [comment]
Hey there, I just released version [2.2.1 Beta 3](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/2.2.1-Beta-3), which will hopefully fix these issues.

Let me know if you experience more problems!
